libsolidity/codegen/YulUtilFunctions.h

```diff
@@ -126,6 +126,12 @@ class YulUtilFunctions
 	/// signature: (start, end)
 	std::string clearStorageRangeFunction(Type const& _type);
 
+	/// @returns the name of a function that will clear the memory area given
+	/// by the start and end (exclusive) parameters (bytes). Only works
```

Comment:
What do we need the type for? Should mention that it only works for `(end - start)` being a multiple of 32

Reply:
I removed it for now as `delete` doesn't actually need this function